build.sbt

```diff
@@ -8,7 +8,12 @@ organization := "io.sphere"
 
 version := "1.0-SNAPSHOT"
 
-val sunriseDesignVersion = "0.31.0"
+lazy val sunriseDesignVersion = "0.34.0"
+
+lazy val sphereJvmSdkVersion = "1.0.0-M21-2015-11-13-11-39-45-facets-SNAPSHOT"
```

Comment:
Thanks, I also planned to move the frequent changing stuff up.
Can we use here the M22-SNAPSHOT like in my branch? Otherwise the master won't compile anymore.